src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/anonymization/SpecialAnonymizationUserPage.java

```diff
@@ -0,0 +1,57 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.special.anonymization;
+
+import com.wikia.webdriver.common.contentpatterns.URLsContent;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.special.SpecialPageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.special.renametool.HelpPage;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.special.renametool.HelpPage'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)